db/common/ddl/urbo_categories_ddl.sql

```diff
@@ -249,7 +249,7 @@ CREATE OR REPLACE FUNCTION urbo_pk_qry(
     FOREACH _tb IN ARRAY _tb_arr
       LOOP
         _stm = format(
-          'ALTER TABLE %s ADD COLUMN id bigserial NOT NULL;
+          'ALTER TABLE %s ADD COLUMN IF NOT EXISTS id bigserial NOT NULL;
```

Comment:
`IF NOT EXIST` in the `ALTER TABLE`'s `ADD COLUMN` is only available from PG 9.6. Currently, the PG in Urbo's CARTO is 9.5.2, and I don't really know when is going to be upgraded.

Reply:
This was causing problems when running function ``cartofunctions`` as step of ``deploy.sh``.

Currently, all psql deployments (prod, dev) use PG 9.6.

I have confirmed that carto accounts ``smart-admin``, ``cedus-admin`` now uses PG 10. ``urbo-production`` still uses 9.5.2. Damn. I wil revert the changes.